jenkins/vs/buildimage-vs-image-released-pr/Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 pipeline {
-    agent { node { label 'jenkins-kvm-workers' } }
+    agent { node { label 'jenkins-build-workers' } }
```

Comment:
we need to separate this job like what we did for builimage-vs-image before we change the tag.